docs/getting-started/config-examples/application-download-part3-custom-webservice.conf

```diff
@@ -0,0 +1,187 @@
+global {
+  spark-options {
+    "spark.hadoop.javax.jdo.option.ConnectionURL" = "jdbc:derby://metastore:1527/db;create=true"
+    "spark.hadoop.javax.jdo.option.ConnectionDriverName" = "org.apache.derby.jdbc.ClientDriver"
+    "spark.hadoop.javax.jdo.option.ConnectionUserName" = "sa"
+    "spark.hadoop.javax.jdo.option.ConnectionPassword" = "1234"
+    "spark.sql.shuffle.partitions" = 2
+    "spark.databricks.delta.snapshotPartitions" = 2
+  }
+}
+
+dataObjects {
+  ext-departures {
+    type = CustomWebserviceDataObject
+    schema = """array< struct< icao24: string, firstSeen: integer, estDepartureAirport: string, lastSeen: integer, estArrivalAirport: string, callsign: string, estDepartureAirportHorizDistance: integer, estDepartureAirportVertDistance: integer, estArrivalAirportHorizDistance: integer, estArrivalAirportVertDistance: integer, departureAirportCandidatesCount: integer, arrivalAirportCandidatesCount: integer >>"""
+    baseUrl = "https://opensky-network.org/api/flights/departure"
+    nRetry = 5
+    queryParameters = [{
+      airport = "LSZB"
+      begin = 1630200800
+      end = 1630310979
+    },{
+      airport = "EDDF"
+      begin = 1630200800
+      end = 1630310979
+    }]
+    timeouts {
+      connectionTimeoutMs = 200000
+      readTimeoutMs = 200000
+    }
+  }
+  stg-departures {
+    type = JsonFileDataObject
+    path = "~{id}"
+  }
+  int-departures {
+    type = DeltaLakeTableDataObject
+    path = "~{id}"
+    table {
+      db = "default"
+      name = "int_departures"
+      primaryKey = [icao24, estdepartureairport, dt]
+    }
+  }
+
+  ext-airports {
+    type = WebserviceFileDataObject
+    # uses redirects to the URL below
+    url = "https://ourairports.com/data/airports.csv"
+    # url = "https://davidmegginson.github.io/ourairports-data/airports.csv"
+    followRedirects = true
+    readTimeoutMs=200000
+  }
+  stg-airports {
+    type = CsvFileDataObject
+    path = "~{id}"
+  }
+  int-airports {
+    type = DeltaLakeTableDataObject
+    path = "~{id}"
+    table {
+      db = "default"
+      name = "int_airports"
+      primaryKey = [ident]
+    }
+  }
+
+  btl-departures-arrivals-airports {
+    type = DeltaLakeTableDataObject
+    path = "~{id}"
+    table {
+      db = "default"
+      name = "btl_departures_arrivals_airports"
+    }
+  }
+
+  btl-distances {
+    type = DeltaLakeTableDataObject
+    path = "~{id}"
+    table {
+      db = "default"
+      name = "btl_distances"
+    }
+  }
+
+}
+
+actions {
+  //download-departures {
```

Comment:
why is this commented out? We are supposed to execute the feed download-departures at the end of the first part of part 3

Reply:
done.